### DISCUSSION TITLE
[FEATURE] item lvl (drop) scale alike VAS autobalance

### DISCUSSION THREAD

**Discussion by wampirr** [azerothcore/azerothcore-wotlk#3922] [Ideas]
Hello comrades!

I have a fixed idea about item lvl scaling(ofc stats mainly) =)
Scale drop from BlackTemple same way as creeps are scaled there with VAS Autobalance.
Or any other way like editing db values once and for all to the exact level (db script, module or anything).
Feeling challenging accuracy needed with this formula in db dcript...
If it already exists, would you pls share?

Thanks again for the project and comunity!

**Comment by wampirr** [comment]
It's ok. =)
Does anyone have at least 1 or 2 abstract formulas of scaling?
I can't find them. (Special thanks to retarded scale in the last patches...)

**Comment by Viste** [comment]
WowD(mangos trinity and others) base were issues with the design of the software which could not be fixed easily in the codebase - like making the server scalable.
